packages/webdriverio/src/middlewares.js

```diff
@@ -13,6 +14,9 @@ export const elementErrorHandler = (fn) => (commandName, commandFn) => {
         return fn(commandName, async function elementErrorHandlerCallbackFn () {
             const element = await implicitWait(this, commandName)
             this.elementId = element.elementId
+            if(this.isW3C) {
+                this[ELEMENT_KEY] = element.elementId
+            }
```

Comment:
There is no need to avoid setting the key for non-W3C browser given that they aren't many people running those type of tests anymore.

```suggestion
            this[ELEMENT_KEY] = element.elementId
```

Reply:
fixed

Reply:
@christian-bromann Oops, looks like this fix breaks a unit test. Should I revert it?